pkg/controller/account/ec2.go

```diff
@@ -168,24 +182,96 @@ func (r *ReconcileAccount) InitializeRegion(
 	}
 
 	err = r.BuildAndDestroyEC2Instances(reqLogger, account, awsClient, instanceInfo, managedTags, customerTags)
-
 	if err != nil {
 		createErr := fmt.Sprintf("Unable to create instance in region: %s", region)
 		controllerutils.LogAwsError(reqLogger, createErr, nil, err)
 		// Notify Error channel that this region has errored and to move on
 		ec2Errors <- regionInitializationError{ErrorMsg: createErr, Region: region}
-
 		return err
 	}
 
 	successMsg := fmt.Sprintf("EC2 instance created and terminated successfully in region: %s", region)
 
+	// if vpcID exists then make sure we delete it
+	if sampleVPCID != "" {
+		err = deleteVpc(reqLogger, awsClient, sampleVPCID)
+		if err != nil {
+			deleteVpcErr := fmt.Sprintf("Unable to delete VPC with ID: %s", sampleVPCID)
+			controllerutils.LogAwsError(reqLogger, deleteVpcErr, nil, err)
```

Comment:
I'm torn on if we want to error out if the VPC can't be deleted (especially if we add retry logic) or if we want to just push on.  I don't have an answer, I just want to call this out as something to think about.

Reply:
IMO we should error out. If we cant delete the VPC we should not finish initialization